package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "fitvids",
+  "version": "1.1.0",
+  "description": "FitVids makes video embeds responsive",
+  "keywords": [
+    "FitVids",
+    "Responsive",
+    "RWD",
+    "YouTube",
+    "Vimeo"
+  ],
+  "license": "WTFPL",
+  "main": "jquery.fitvids.js",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "components",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {},
+  "devDependencies": {}
```

Comment:
remove these two lines. no deal with empty stuff. if npm needs them, it will include it again.